src/strings/PowerFxResources.en-US.resx

```diff
@@ -6549,4 +6549,8 @@
     <value>value</value>
     <comment>function_parameter - Second argument to the Set function - any Power Apps value.</comment>
   </data>
+  <data name="ErrExpectedIdentifierArg_Name" xml:space="preserve">
```

Comment:
+ @CarlosFigueiraMSFT 

Files other than en-US should not be updated in feature PRs. The translations are done by the localization team on a regular basis.